pyartcd/pyartcd/pipelines/buils_sync.py

```diff
@@ -0,0 +1,92 @@
+import time
+import click
+import logging
+from pyartcd import constants, exectools
+from pyartcd.cli import cli, click_coroutine, pass_runtime
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@cli.group("imagestreams", short_help="Manage ART equivalent images in upstream CI.")
+async def build_sync()
+    pass
+
+
+@build_sync.command('backup', short_help='Reads streams.yml and mirrors out ART equivalent images to api.ci.')
+@pass_runtime
+@click_coroutine
+async def backup_imagestream():
+    allNameSpaces = ["ocp", "ocp-priv", "ocp-s390x", "ocp-s390x-priv", "ocp-ppc64le", "ocp-ppc64le-priv", "ocp-arm64", "ocp-arm64-priv"]
+    tar = tarfile.open("app.ci-backup.tgz","w:gz")
+    for ns in allNameSpaces:
+        cmd = f"oc --kubeconfig {constants.CIKUBECONFIG} get is -n {ns} -o yaml"
```

Comment:
If possible, use an array instead. There will be issues if there is a space in any argument.